src/abs.rs

```diff
@@ -279,8 +279,22 @@ impl Deref for PathAbs {
     }
 }
 
-impl Into<PathArc> for PathAbs {
-    fn into(self) -> PathArc {
-        self.0
+impl From<PathAbs> for PathArc {
```

Comment:
This is good, as ["`From<T> for U implies Into<U>for T`"](https://doc.rust-lang.org/std/convert/trait.From.html)